test_cairo_contracts/compiled_contracts/valid_program.json

```diff
@@ -0,0 +1,1370 @@
+{
```

Comment:
Don't commit this file.
It's should be generated by the executable, not provided in the repo